rl_and_pong/pong_model_container.py

```diff
@@ -30,7 +30,7 @@ def f():
         self.agent.restore(path)
         # Run test prediction to load the model
         print("Predicted {} in constructor".format(
-            self.agent.compute_action(np.random.random(8))))
+            self.agent.compute_action(np.random.random(7))))
```

Comment:
You can just comment out this whole print statement. The version I pushed to Dockerhub does that (see #36).